packages/api/src/brevo-config/brevo-config.resolver.ts

```diff
@@ -93,8 +115,19 @@ export function createBrevoConfigResolver({
                 validateNotModified(brevoConfig, lastUpdatedAt);
             }
 
+            if (!input.senderMail || !input.senderName) {
+                throw new Error("Sender mail, sender name are required");
+            }
```

Comment:
This can be removed as wenn right? Since it is required in the input?

Reply:
Removed here: https://github.com/vivid-planet/comet-brevo-module/pull/113/commits/75d68ae157b12187ca0775b0978795a82f4117e4